t04_refactored.py

```diff
@@ -197,9 +197,63 @@ def team_11_adv(username, delay):
 
 
 def team_12_adv(username, delay):
-    pass
-    # TODO Add your code here
+    """
+    https://docs.google.com/document/d/1-VIVi-Txn87fIW0pf9LwcYZbWpGTJC0tbpnC-va0IBo/edit?usp=sharing
+    Besher Kitaz
+    Feda Mohammadi
+    :return: None
+    """
+    dead = False
+    delay = 1.0
```

Comment:
`delay` is an input parameter now, so remove this line. It is overwriting the parameter, rendering it obsolete.